### DISCUSSION TITLE
browser closed while downloading and file won't open.

### DISCUSSION THREAD

**Discussion by jlog2507** [aclap-dev/video-downloadhelper#66] [Help and Q&A]
### Your question

I accidently closed browser(firefox) while downloading a streaming video and the video file(mp4) won't open.  Usually when closing browser while downloading the downloading automatically stop and file would work just fine. I guess setting are little different with my new computer than my old one. Anyways, are there ways to fix the file?

**Comment by GrampaWildWilly** [comment]
I'd say your chances are very low.  Try running ffprobe against the file.  There's an ffprobe included with the CoApp.  Just ffrpobe "file name".  That's it.  Post the results here.  I suspect it will say something about a missing moov atom.  If that's the error, you're out of luck.  For future reference, you can go into VDH & stop a download, & that should leave you with a functional downloaded file.  That's a graceful, although premature, termination.  But abrupt, unexpected events like closing the browser or a crash are unlikely to give you results you can use.  Usually, closing the browser does not gracefully terminate the download, contrary to what you said.  It ungracefully kills the download.  Don't rely on that to ever work.

**Reply by jlog2507** [reply]
Just as you suspected. thank you for the reply tho.
